docs/src/modules/java/pages/index.adoc

```diff
@@ -1,33 +1,42 @@
-= Developing with Java
+= Developing with Java or Scala
+:page-supergroup-java-scala: Language
 
 include::ROOT:partial$include.adoc[]
 include::partial$attributes.adoc[]
 
-The Akka Serverless Java SDK guides you through implementing components matching the protobuf APIs you design. This page describes prerequisites for Java development and basic requirements for a development project.
+Select your preferred language (Java/Scala) above.
 
+The Akka Serverless [.group-java]#Java# [.group-scala]#Scala# SDK guides you through implementing components matching the protobuf APIs you design. This page describes prerequisites for [.group-java]#Java# [.group-scala]#Scala# development and basic requirements for a development project.
+
+[.group-java]
 NOTE: Lightbend provides Tier 1 support for the Java SDK. See https://developer.lightbend.com/docs/akka-serverless/faq/index.html#_what_languages_are_supported[an explanation of support tiers] for more information.
 
-Your development project needs to include the Akka Serverless Java SDK and logic to start the gRPC server. You define your components in gRPC descriptors and use `protoc` to compile them. Finally, you implement business logic for service components.
+Your development project needs to include the Akka Serverless [.group-java]#Java# [.group-scala]#Scala# SDK and logic to start the gRPC server. You define your components in gRPC descriptors and use `protoc` to compile them. Finally, you implement business logic for service components.
 
-To save the work of starting from scratch, the Java xref:java:quickstart-java-maven.adoc[code generation tool] creates a project via a Maven archetype, complete with descriptors and implementations. Or, you can start from one of our fully implemented https://developer.lightbend.com/docs/akka-serverless/quickstart/index.html[Quickstart applications].
+To save the work of starting from scratch, the Java xref:java:quickstart-template.adoc[code generation tool] creates a project from a template, complete with descriptors and implementations. Or, you can start from one of our fully implemented https://developer.lightbend.com/docs/akka-serverless/quickstart/index.html[Quickstart applications].
 
 == Prerequisites
 
 The following are required to develop services in Java:
 
 Java:: Akka Serverless requires at least Java {minimum-java-version}, though we recommend using Java {recommended-java-version}, which has better support for running in containers.
 
+[.group-java]
```

Comment:
TODO ofc we want to use tabs instead of groups in these cases, will update later